core/src/franchise/player/player.service.ts

```diff
@@ -94,4 +123,353 @@ export class PlayerService {
+                    //             embeds: [ {
+                    //                 title: "Rankdown Available",
+                    //                 description: `You have been offered a rankout from ${player.skillGroup.profile.description} to ${skillGroup.profile.description}.`,
+                    //                 author: {
+                    //                     name: `${orgProfile.name}`,
+                    //                 },
+                    //                 fields: [
+                    //                     {
+                    //                         name: "New League",
+                    //                         value: `${skillGroup.profile.description}`,
+                    //                     },
+                    //                     {
+                    //                         name: "New Salary",
+                    //                         value: `${playerDelta.rankout.salary}`,
+                    //                     },
+                    //                 ],
+                    //                 footer: {
+                    //                     text: orgProfile.name,
+                    //                 },
+                    //                 timestamp: Date.now(),
+                    //             } ],
+                    //             components: [ {
+                    //                 type: ComponentType.ACTION_ROW,
+                    //                 components: [
+                    //                     {
+                    //                         type: ComponentType.BUTTON,
+                    //                         style: ButtonComponentStyle.LINK,
+                    //                         label: "Accept it here!",
+                    //                         url: `${config.web.url}/notifications/${notifId}`,
+                    //                     },
+                    //                 ],
+                    //             } ],
+                    //         },
+                    //         brandingOptions: {
+                    //             organizationId: player.member.organization.id,
+                    //             options: {
+                    //                 author: {
+                    //                     icon: true,
+                    //                 },
+                    //                 color: true,
+                    //                 thumbnail: true,
+                    //                 footer: {
+                    //                     icon: true,
+                    //                 },
+                    //             },
+                    //         },
+                    //     },
+                    // });
+                }
+            } else {
+                await this.updatePlayerStanding(playerDelta.playerId, playerDelta.newSalary);
+            }
+        }))));
+    }
+
+    mle_nextLeague(league: League, dir: -1 | 1): League {
```

Comment:
MLE methods in here must be done to avoid 20 hours of figuring out circular dependency errors. If you don't want it here, you fix it. :)